app/src/main/java/com/example/campusguide/utils/BuildingHighlights.kt

```diff
@@ -33,4 +37,12 @@ class BuildingHighlights(private val googleMap: GoogleMap) {
         }
         return latLngs
     }
+
+    private fun applyPolygonStyle(polygon: PolygonOptions){
+        val primaryColor = ContextCompat.getColor(context, R.color.colorPrimary)
+        val darkColor = ContextCompat.getColor(context, R.color.colorPrimaryDark)
+        polygon.strokeWidth(3.5F)
+        polygon.strokeColor(darkColor)
+        polygon.fillColor(ColorUtils.setAlphaComponent(primaryColor, 40))
+    }
```

Comment:
Could it be worth it to define the z index?

Reply:
Yes, maybe once we have more stuff on the map overlay